test/lambda/trans-setDefaultName.test.js

```diff
@@ -87,6 +87,7 @@ tape.test('lambda instrumentation does not break trans.setDefaultName', function
         cb(null, 'hi')
       })
 
+      server.clear()
```

Comment:
REVIEW NOTE: This was an unrelated fix for this test. It wasn't testing what it thought it was testing and was passing by accident.